test/PowerShellEditorServices.Test/PowerShellEditorServices.Test.csproj

```diff
@@ -104,7 +104,11 @@
   <ItemGroup>
     <Service Include="{82A7F48D-3B50-4B1E-B82E-3ADA8210C358}" />
   </ItemGroup>
-  <ItemGroup />
+  <ItemGroup>
+    <Content Include="NuGet.exe">
```

Comment:
The solution already has NuGet.exe in the root \.nuget folder so you could probably just use that directly rather than including another one here.  Might not be necessary at all if we pull in the reference assemblies as NuGet dependencies for the test project